runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useUserSearch.ts

```diff
@@ -67,14 +72,17 @@ const useUserSearch = (isActivatedUsersOnly: boolean, pageSize = 10, page = 0) =
         if (search !== event.target.value) setSearch(event.target.value);
     };
 
-    const refreshSearch = () => {
+    const refreshSearchNotActivated = () =>
+        setRefreshTrigger(!refreshTrigger);
+
+    const refreshSearchActivated = () =>
         setRefreshTrigger(!refreshTrigger);
-    };
```

Comment:
I'm using that hook for both activated and not activated users in DeleteUserDialog component and by creating two versions of function (first for activated, second for not activated) I can't use one refresh if that component don't know on what page was open, so that is why I have to run both refreshes.